diff --git a/environment/Section 635 2 Criminal Trespass.htm b/environment/Section 635 2 Criminal Trespass.htm
index 0cf938e..17e8598 100644
--- a/environment/Section 635 2 Criminal Trespass.htm	
+++ b/environment/Section 635 2 Criminal Trespass.htm	
@@ -19,8 +19,8 @@ CRIMINAL CODE</b></span></h1>
 UNAUTHORIZED ENTRIES</b></span></h2>
 <h3 style="margin: 0.0px 0.0px 14.0px 0.0px; text-align: center; font: 14.0px Times; color: #000000; -webkit-text-stroke: #000000"><span class="s1"><b>Section 635:2</b></span></h3>
 <p class="p4"><span class="s1">   <b> 635:2 Criminal Trespass. –</b><span class="Apple-converted-space"> </span></span></p>
-<p class="p4"><span class="s1">I. A person is guilty of criminal trespass if, knowing that he is not licensed or privileged to do so, he enters or remains in any place.<span class="Apple-converted-space"> </span></span></p>
-<p class="p4"><span class="s1">II. Criminal trespass is a misdemeanor for the first offense and a class B felony for any subsequent offense if the person knowingly or recklessly causes damage in excess of $1,500 to the value of the property of another.<span class="Apple-converted-space"> </span></span></p>
+<p class="p4"><span class="s1">I. A person is guilty of criminal trespass if, knowing that he is not licensed or privileged to do so, he or she enters or remains in any place where his or her presence would cause harm.<span class="Apple-converted-space"> </span></span></p>
+<p class="p4"><span class="s1">II. Criminal trespass is a misdemeanor for the first offense and a class B felony for any subsequent offense if the person knowingly or recklessly causes damage in excess of $1,500 to the value of the property of another, except in the case where the property in question is fossil fuel infrastructure.<span class="Apple-converted-space"> </span></span></p>
 <p class="p4"><span class="s1">III. Criminal trespass is a misdemeanor if:<span class="Apple-converted-space"> </span></span></p>
 <p class="p4"><span class="s1">(a) The trespass takes place in an occupied structure as defined in RSA 635:1, III; or<span class="Apple-converted-space"> </span></span></p>
 <p class="p4"><span class="s1">(b) The person knowingly enters or remains:<span class="Apple-converted-space"> </span></span></p>
